### ISSUE TITLE
a11y | tab order

### ISSUE THREAD

**Issue by scottqueen-bixal** [GSA/px-bears-drupal#508]
11 | 08/31 | Amy | “Continue” button skipped in keyboard order. | Anyone using a keyboard only or filling out the form without a mouse should be able to complete all actions in the form. | I am unable to use my keyboard to select “continue”. Focus skips from “back” to “top” and there isn’t any way to “continue” using my keyboard. WCAG 2.1.1 | Open the “about the deceased” page. Without using a mouse, tab through each element and fill out the form. Then try to click “continue”. Used Edge via VDI. | “About the deceased” page.See screen shot. | Development | Blank |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

**Comment by angelina-smith** [comment]
Upon re-audit of the updated MVP, keyboard tab order is logical and all interactive elements are accessible via keyboard.
